src/domains/login/common/components/EmailInput.jsx

```
@@ -3,17 +3,27 @@ import LabelWithInput from '../../../../components/web/LabelWithInput'
 import Button from '../../../../components/web/Button'
 import DomainSelector from './DomainSelector'
 import OTPInput from '../../../Find/common/components/OTPInput'
-import { postJoinCheckEmail, postJoinCheckEmailNum } from '../../services/useJoinServices'
-import ToastService from '../../../../utils/toastService'
+import CustomDomain from './CustomDomain'
 
-const EmailInput = ({ register, errors, watch, setValue, trigger, setValidationStatus }) => {
+const EmailInput = ({
+  register,
```

ESLint

[error] 'register' is missing in props validation

(react/prop-types)